ls1intum/Artemis
# `General`: Fix display of no submission after due date

### Checklist
#### General


#### Client

### Motivation and Context


Fix a bug where nothing was displayed after starting an exercise without submitting anything and waiting for the due date to pass.

### Description

This Pr fixes he bug and now shows correctly "No submission" in this case.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 non-Quiz Exercise with a due date

1. Log in to Artemis
2. Navigate to Course Administration
3. Start the exercise but do not submit anything
4. Check that the exercise row correctly displays "No submission"
5. Wait for the due date to pass
6. Check again that the exercise row still correctly displays "No submission"

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


![image](https://github.com/ls1intum/Artemis/assets/38322605/eb6360c4-b04e-4d71-9d29-3876cd233b85)
